app/model/editions/templates/AmericanEdition.scala

```diff
@@ -56,13 +56,11 @@ object AmericanEdition {
       // Crosswords
       FrontCrosswords -> Daily(),
     ),
-    capiQueryPrefillParams = CapiQueryPrefillParams(
-      timeWindowConfig = CapiTimeWindowConfigInDays(
-        startOffset = 0,
-        endOffset = 0,
-        useDate = UseDateQueryParamValue.NewspaperEdition
-      )
+    timeWindowConfig = CapiTimeWindowConfigInDays(
+      startOffset = 0,
+      endOffset = 0,
     ),
+    capiDateQueryParam = CapiDateQueryParam.NewspaperEdition,
     zoneId = ZoneId.of("Europe/London"),
```

Comment:
shouldn't `capiDateQueryParam` be part fo time window class ? it is used for time window anyway (they are used together)
I think that was the initial idea before

Reply:
After consultation with David, it had to be taken out.
useDate is specific to Issue, but time window can be specific to Issue, Front or Collection.
We don't want people to be able to set a different useDate at different points.